pkgs/development/python-modules/ailment/default.nix

```diff
@@ -28,5 +29,7 @@ buildPythonPackage rec {
     homepage = "https://github.com/angr/ailment";
     license = with licenses; [ bsd2 ];
     maintainers = with maintainers; [ fab ];
+    # pyyex is not available for aarch64
+    broken = stdenv.isAarch64;
```

Comment:
this isn't really needed since pyvex is already marked as unsupported

Reply:
Removed, thanks